plugins/autopeering/local/local.go

```diff
@@ -78,6 +70,10 @@ func configureLocal() *peer.Local {
 	// key, _ := peerDB.LocalPrivateKey()
 	// fmt.Printf("Seed: base58:%s\n", key.Seed().String())
 
+	// TODO: remove requirement for PeeringKey in hive.go
```

Comment:
`NewLocal` requires a peering service to be present during construction. As now the identity is independent of the auto peering, this requirement should be removed in `hive.go`. However, due to version conflicts, dummy this dummy solution first seems easier.